### PR TITLE
Repackage Windows Security Events

### PR DESCRIPTION
**Pull request by v-atulyadav**
Required items, please complete
   
   Change(s):
   - Included Sub-Techniques in Template
   - Shorten the hunting query description

   Reason for Change(s):
   - Repackage for sub-technique inclusion

   Version Updated:
   - Yes

   Testing Completed:
   - Yes
   
[WindowsSecurityEvents.docx](https://github.com/Azure/Azure-Sentinel/files/14021409/WindowsSecurityEvents.docx)

### CODE REVIEW

**Review by v-rusraut — DISMISSED** [review]

**Review by v-rusraut — DISMISSED** [review]

File: Solutions/Windows Security Events/Hunting Queries/Suspicious_Windows_Login_outside_normal_hours.yaml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 id: e7bfbc3f-98c7-4aaa-a64c-de9c058b86b2
 name: Suspicious Windows Login Outside Normal Hours
 description: |
+  'Query identifies unusual logon events outside a user's normal hours by comparing with the last 14 days, flagging anomalies based on historical patterns.'
```
Comment (rahul0216) [inline Solutions/Windows Security Events/Hunting Queries/Suspicious_Windows_Login_outside_normal_hours.yaml:4]: Query identifies unusual logon events outside a user's normal hours by comparing with the last 14 days' logon activity, flagging anomalies based on historical patterns.

File: Solutions/Windows Security Events/Hunting Queries/masquerading_files.yaml [hunk 2]
```diff
@@ -1,6 +1,8 @@
 id: 60304ebf-ebdd-4869-a702-e0216d90ab46
 name: Masquerading files
 description: |
+  'Malware often mimics system process names like svchost.exe to hide. Analysts should query for svchost.exe, excluding known SIDs and legitimate paths.'
```
Comment (rahul0216) [inline Solutions/Windows Security Events/Hunting Queries/masquerading_files.yaml:4]: Malware writers often use windows system process names like svchost.exe to hide malicious activities. Query searches for execution of process svchost.exe, filtering out execution by well-known SIDs and from legitimate paths.

File: Solutions/Windows Security Events/Hunting Queries/persistence_create_account.yaml [hunk 3]
```diff
@@ -1,6 +1,8 @@
 id: 5e76eaf9-79a7-448c-bace-28e5b53b8396
 name: Summary of users created using uncommon/undocumented commandline switches
 description: |
+  'Summarizes uses of uncommon & undocumented commandline switches to create persistence. User accounts may be created to achieve persistence on a machine.'
```
Comment (rahul0216) [inline Solutions/Windows Security Events/Hunting Queries/persistence_create_account.yaml:4]: Summarizes uses of uncommon & undocumented commandline switches to create user accounts. User accounts may be created to achieve persistence on a machine.

File: Solutions/Windows Security Events/Hunting Queries/uncommon_processes.yaml [hunk 4]
```diff
@@ -1,6 +1,8 @@
 id: 2ff4b10c-7056-4898-83fd-774104189fd5
 name: Uncommon processes - bottom 5%
 description: |
+  'Query highlights first-time rare processes, best over 3+ days, to flag new, potentially unauthorized binaries in stable environments for attack detection.'
```
Comment (rahul0216) [inline Solutions/Windows Security Events/Hunting Queries/uncommon_processes.yaml:4]: Query highlights uncommon, rare process runs, to flag new potentially unauthorized binaries in stable environments for potential attack detection.

**Review by rahul0216 — COMMENTED** [review]
Please see the inline comments, suggested descriptions.

**Review by rahul0216 — APPROVED** [review]
Description changes look good.

**Review by v-rusraut — APPROVED** [review]

**Pull request merged**